src/main/java/teammates/common/util/Const.java

```diff
@@ -986,7 +986,7 @@ private Const() {
         public static final String INSTRUCTOR_STUDENTS = "/instructor/students";
         public static final String COURSE_STUDENT_DETAILS = "/courses/students/details";
         public static final String STUDENT_COURSE = "/student/course";
-        public static final String STUDENT_PROFILE_PICTURE = "/students/profilePic";
+        public static final String STUDENT_PROFILE_PICTURE = "/student/profilePic";
```

Comment:
Should probably standardize the name: `_` implies a `/`.
```suggestion
        public static final String STUDENT_PROFILEPIC = "/student/profilePic";
```

Reply:
I really don't think this matters.

However, if the rest feel the same way I can change it :)